helpers/announce.js

```diff
@@ -22,27 +20,35 @@ export function announce(message) {
 		document.body.appendChild(container);
 	}
 
-	/* Need to give browser enough time to create the live region so that it will
-	treat the change as an update. Firefox sometimes ignores changes if the region
-	and update are made too quickly in succession. RequestAnimationFrame is not
-	sufficient here. Also, for some strange reason, sometimes VO will not announce
-	duplicate messages even if we remove the child so we also append a non-breaking space. */
-	const elem = [...container.childNodes].find((c) => c.textContent === message);
-	if (elem) {
-		elem.parentNode.removeChild(elem);
-		message = message.concat('\u00A0');
```

Comment:
I tested out the PR site and it looks like the defect that made this necessary still exists (On Chrome + VoiceOver). If you have identical messages, for example by putting "Hello" in both inputs in the demo, it will only announce the first one.

We added this non-breaking space so that the screen-reader will perceive identical messages differently and announce them.

Reply:
@GZolla I don’t think the change solves the problem with the repeated messages.

What the code was doing previously was checking if the last message was equal to the current message and if it was it adds the space. So if you have repeated messages, it would look something like this:
```
Hello // Not equal to last message so prints as usual
Hello\u00A0 // Equal to last message so adds a space
Hello // Not equal to last message so does not add a space
Hello\u00A0 ...
```

In your code, it adds a space if the message is in the list so this will read the first 2 messages but not subsequent messages:
```
Hello
Hello\u00A0
Hello\u00A0 // Identical to previous message, so VO will not read this
Hello\u00A0 // Identical to previous message, so VO will not read this
```

So I think we should be checking the equality of the current and previous messages for this to work.